### ISSUE TITLE
Hidden checkbox with missing label on the cookie settings page

### ISSUE THREAD

**Issue by andybroomfield** [localgovdrupal/localgov_eu_cookie_compliance#3]
Being flagged up on acessbility.
On the cookies page, there are hidden checkboxes that do not have a label.
We should either not render the checkboxes, or if we need them visible, hide them from screenreaders.

<img width="473" alt="Screenshot 2023-02-24 at 4 44 53 pm" src="https://user-images.githubusercontent.com/1467480/221237934-aeac329a-62a1-4203-8c2b-cad9aae49dbc.png">

**Comment by keelanfh** [comment]
We addressed this one with `display: none` in CSS but it might be more robust to address it in the HTML in this module if possible, so that it would not be dependent on people implementing a specific theme

https://github.com/essexcountycouncil/essex-gov-uk-drupal/blob/develop/web/modules/custom/ecc_cookie_compliance/css/ecc_cookie.css#L1-L3

**Comment by keelanfh** [comment]
I'm wondering if this is a false positive in WAVE. I think this module (through jQuery) sets `display: none` on the element, it doesn't appear in the accessibility tree and isn't read out by screenreaders, but WAVE is still flagging it.

![image](https://github.com/localgovdrupal/localgov_eu_cookie_compliance/assets/19519457/54e278d4-3158-411a-bf35-b5cc58b54c8c)

**Comment by keelanfh** [comment]
Just tested with Axe and Siteimprove browser extensions and neither picked this up - but they did when `display: none;` was removed

**Comment by keelanfh** [comment]
And on WAVE's website, they confirm that this is by design: https://wave.webaim.org/help#hidden. I'm going to close this issue as, unlike the issue in the banner itself, I don't think there's a genuine accessibility issue here.
